test/e2e/lifecycle/framework.go

```diff
@@ -16,9 +16,102 @@ limitations under the License.
 
 package lifecycle
 
-import "github.com/onsi/ginkgo"
+import (
+	"fmt"
+	"os/exec"
+	"path"
+	"strings"
+	"time"
+
+	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
+
+	"github.com/onsi/ginkgo"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apimachinery/pkg/version"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
 
 // SIGDescribe annotates the test with the SIG label.
 func SIGDescribe(text string, body func()) bool {
 	return ginkgo.Describe("[sig-cluster-lifecycle] "+text, body)
 }
+
```

Comment:
Why do you move these functions into `test/e2e/lifcycle/`?
These functions are used in ` test/e2e/cloud/gcp` only, so these functions should be moved to ` test/e2e/cloud/gcp` I think according to https://github.com/kubernetes/kubernetes/issues/86052

Reply:
Sure